addon/components/fixtable-grid.js

```diff
@@ -51,16 +51,19 @@ export default Ember.Component.extend({
 
   notifyReloadContent() {
     let handler = this.get('onReloadContent');
-    if (typeof handler !== 'function') { return; }
+    if (typeof handler === 'function') {
+      let sortBy = this.get('sortBy');
+      let sortInfo = sortBy ? { key: sortBy, ascending: this.get('sortAscending') } : null;
 
-    let sortBy = this.get('sortBy');
-    let sortInfo = sortBy ? { key: sortBy, ascending: this.get('sortAscending') } : null;
+      handler(
+        this.get('currentPage'),
+        this.get('pageSize'),
+        this.get('filterToApply') || {},
+        sortInfo);
+    }
 
-    handler(
-      this.get('currentPage'),
-      this.get('pageSize'),
-      this.get('filterToApply') || {},
-      sortInfo);
+    // if we need to reload content, that also means the selection will be reset
+    this.notifyRowSelectionChanged([]);
```

Comment:
I'm guessing this was an explicit decision to NOT track possible duplicates between selected rows BEFORE `notifyReloadContent` and AFTER, leaving the responsibility / choice to reselect them to the parent container - ?